adapters/src/main/java/com/squareup/moshi/adapters/RuntimeJsonAdapterFactory.java

```diff
@@ -0,0 +1,143 @@
+package com.squareup.moshi.adapters;
```

Comment:
nit: copyright header. If this is derived from Gson’s, copy over that copyright header?

Reply:
thanks, done!